warre/acct/login.html

```diff
@@ -27,9 +27,9 @@
 	<link rel="stylesheet" href="login.css">
 
 	<!-- internal resource links -->
-	<script src="/js/global_on_page_load.js"></script>
-	<script src="/js/banner.js"></script>
-	<script src="/js/authentication.js"></script>
+	<script src="../js/global_on_page_load.js"></script>
```

Comment:
Shouldn't this be the same behavior as before? I think `/` resolves to the root of the web server, which should be correct in the old code. Was there an error loading these?

Reply:
There was an error when I tried it